.github/CODEOWNERS

```
@@ -1,2 +1,2 @@
-* @elastic/platform-docs
+* @elastic/docs
 /.github/workflows/co-docs-builder.yml @elastic/docs-engineering
```

this will tag all of docs on all PRs. until we move the rest of the docs into this repo I think we want this:
```suggestion
* @elastic/platform-docs
/.github/workflows/co-docs-builder.yml @elastic/docs-engineering
/serverless/serverless-changelog.asciidoc @elastic/docs 
```